interfaces/instance-entries/OWMapControlInterface.ts

```diff
@@ -0,0 +1,6 @@
+export interface OWMapControlInterface {
```

Comment:
[This probably should be using ](https://github.com/ps2alerts/constants/blob/outfit-wars/interfaces/OutfitwarsTerritoryResultInterface.ts)